src/libraries/System.Private.Xml/src/System/Xml/Dom/DocumentXPathNavigator.cs

```diff
@@ -219,6 +223,8 @@ private string ValueText
                            && nextSibling.IsText);
                     value = builder.ToString();
                 }
+                // TODO: Not sure if this is right, maybe we should change to return string.Empty in case of null.
```

Comment:
```suggestion
                // TODO-NULLABLE: Not sure if this is right, maybe we should change to return string.Empty in case of null.
```

Reply:
also might be worth to expand "this". Right now it makes sense since you're working on nullable annotation. Imagine losing history of changes and people getting used to nullability annotations everywhere - "this" is not so obvious in that scenario. Perhaps something like `TODO-NULLABLE: consider returning string.Empty instead of null`